src/App.tsx

```diff
@@ -1,24 +1,215 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { FormEvent, useEffect, useState } from 'react';
+import { Todo } from './Components/Todo';
+import { Todo as TodoType } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  removeTodo,
+  updateTodo,
+} from './api/todos';
+import { Notification } from './Components/Notification';
+import { Filter } from './Components/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10539;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<TodoType[]>([]);
+  const [error, setError] = useState('');
+  const [filteredTodos, setFilteredTodos] = useState<TodoType[]>([]);
+  const [filter, setFilter] = useState('');
+  const [addValue, setAddValue] = useState('');
+  const [isInputDisable, setIsInputDisable] = useState(false);
+  const [tempTodo, setTempTodo] = useState<TodoType | null>(null);
+  const [deleteIds, setDeleteIds] = useState<number[]>([]);
+  const [isUpdating, setIsUpdating] = useState(false);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(response => {
+        setTodos(response);
+      })
+      .catch(() => setError('Unable to Load todos'));
+  }, [deleteIds, isUpdating]);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(response => {
+        setFilteredTodos(response);
+      })
+      .catch(() => setError('Unable to Load todos'));
+  }, [todos, tempTodo]);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setError('');
+    }, 3000);
+  }, [error]);
+
+  useEffect(() => {
+    switch (filter) {
+      case 'Active': {
```

Comment:
Consider adding an enum so that you don't have to use strings in your code
```suggestion
      case FilterType.ACTIVE: {
```